File: Arithmetic_CheckPrimeFast.cs
using System;

namespace Arithmetic_CS
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Welcome to our demonstration sequels");
            Console.WriteLine("Hope you enjoy (and follow) the lessons.");
            Console.WriteLine("\r\n");

            /*
            * Test for the primeness of a number.
            */
            int test_guy = 97;
            CheckPrimeFast fast_check = new CheckPrimeFast(test_guy); // Is 97 is prime number?

            string result = "Prime State:\r\n";
            if (fast_check.verifyPrime())
            {
                result += test_guy + " is a prime number.";
            }
            else
            {
                result += test_guy + " is not a prime number.";
            }

            Console.WriteLine(result);

        }
    }
}

File: Arithmetic_CheckPrimeFast.vb
Module Arithmetic_check_prime_fast

    Sub Main()
        ' Use the check prime fast Class
        Dim test_guy = 98
        Dim prime_check As New CheckPrimeFast
        prime_check._init_(test_guy)

        Dim result = "Prime State:" & Environment.NewLine
        If prime_check.verifyPrime() Then
            result = test_guy & " is a prime number."
        Else
            result += test_guy & " is not a prime number." & Environment.NewLine
            result += "At least one factor of " & test_guy & " is " & prime_check.possible_factor
        End If

        Console.WriteLine(result)

    End Sub

End Module
